x-pack/platform/packages/shared/file-upload-common/src/types.ts

```diff
@@ -19,4 +19,13 @@ export interface OpenFileUploadLiteContext {
   indexSettings?: IndicesIndexSettings;
   autoAddInference?: string;
   autoCreateDataView?: boolean;
+  initialIndexName?: string;
+  flyoutContent?: FlyoutContent;
+}
+
+export interface FlyoutContent {
+  title?: string;
+  description?: string;
+  showFileContentPreview: boolean;
```

Comment:
Should it be optional? 
```suggestion
  showFileContentPreview?: boolean;
```

Reply:
Updated in https://github.com/elastic/kibana/pull/213275/commits/c19ef736fe8052616736eedb6757c01c4c5df97b